processor/src/main/java/org/mapstruct/ap/model/source/builtin/BuiltInMappingMethods.java

```diff
@@ -35,7 +36,6 @@
 
     public BuiltInMappingMethods(TypeFactory typeFactory) {
         builtInMethods = Collections.newArrayList(
-            new JaxbElemToValue( typeFactory ),
             new DateToXmlGregorianCalendar( typeFactory ),
             new XmlGregorianCalendarToDate( typeFactory ),
```

Comment:
You didn't have issues with `XmlGregorianCalendar`? Is this part of the Android platform then?